Library/Formula/jsawk.rb

```diff
@@ -1,16 +1,18 @@
-require 'formula'
-
 class Jsawk < Formula
-  homepage 'https://github.com/micha/jsawk'
-  url 'https://github.com/micha/jsawk/archive/1.4.tar.gz'
-  sha1 '4f2c962c8a5209764116457682985854400cbf24'
+  homepage "https://github.com/micha/jsawk"
+  url "https://github.com/micha/jsawk/archive/1.4.tar.gz"
+  sha1 "4f2c962c8a5209764116457682985854400cbf24"
 
-  head 'https://github.com/micha/jsawk.git'
+  head "https://github.com/micha/jsawk.git"
 
-  depends_on 'spidermonkey'
+  depends_on "spidermonkey"
 
   def install
-    mv "README.markdown", "README"
```

Comment:
Why this change, out of interest? I think this may be there as `README` files get installed to the `prefix` by default but `README.markdown` might not.